.importjs.json

```diff
@@ -2,6 +2,7 @@
   {
     "environments": ["node"],
     "ignorePackagePrefixes": ["lodash."],
+    "logLevel": "debug",
```

Comment:
I remember thinking user-specific config would have been helpful for another thing, but I don't remember what it was anymore. Can you think of other things that user-specific config would be useful for?